<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JsonBuilderMw</class>
 <widget class="QMainWindow" name="JsonBuilderMw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JsonBuilderMw</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="inputFile"/>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="bInput">
      <property name="text">
       <string>input</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="outputFile"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="bOutput">
      <property name="text">
       <string>output</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QTextEdit" name="lines"/>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="bCompile">
      <property name="text">
       <string>compile</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
